apps/angular/src/app/features/home/anime-catalog/components/anime-filter-form/anime-filter-form.component.ts

```diff
@@ -0,0 +1,67 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { MatInputModule } from '@angular/material/input';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { FormsModule } from '@angular/forms';
+import { MatSelectChange, MatSelectModule } from '@angular/material/select';
+import { AnimeType } from '@js-camp/core/models/anime-type';
+import { MatButtonModule } from '@angular/material/button';
+import { MatIconModule } from '@angular/material/icon';
+
+/** Anime selector form. */
+@Component({
+	selector: 'camp-anime-filter-form',
+	standalone: true,
+	imports: [MatInputModule, MatFormFieldModule, FormsModule, MatButtonModule, MatSelectModule, MatIconModule],
+	templateUrl: './anime-filter-form.component.html',
+	styleUrl: './anime-filter-form.component.css',
+	changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AnimeFilterFormComponent {
+
+	/** Selected type by the user. */
+	@Input()
+	public selectedType: AnimeType | null = null;
+
+	/** Searched result by the user. */
+	@Input()
```

Comment:
If any of those inputs are required we can makes the contract between parent and child component more strict
```suggestion
	@Input({ required: true })
```